src/link/data-access-link/useGetLinks.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback, useEffect } from "react";
+import { axiosInstance } from "../../sharing/util";
+import { mapLinksData } from "../../link/util-map/mapLinksData";
+import { useAsync } from "../../sharing/util";
+import { ALL_LINKS_ID } from "./constant";
+
+interface MapDataFormatProps {
+  id: string | number;
+  created_at: Date;
+  url: string;
+  image_source: string;
+  title: string;
+  description: string;
+}
+export const useGetLinks = (folderId = ALL_LINKS_ID) => {
+  const queryString = folderId === ALL_LINKS_ID ? "" : `?folderId=${folderId}`;
+  const getLinks = useCallback(
+    () => axiosInstance.get(`users/1/links${queryString}`),
```

Comment:
💬 지금은 "전체" 에 대한 핸들링이 들어가져있어서 100% 호환되진 않지만, 이런 방식으로도 queryString 핸들링이 가능합니다

```suggestion
    () => axiosInstance.get(`users/1/links`, { params: { folderId } }),
```